pkg/postgres/repository.go

```diff
@@ -399,6 +416,58 @@ ORDER BY estimates.identifier ASC`,
 	return
 }
 
+//GetRecentlyActiveRuns collects all the data about any runs that have been updated
+//since touchThreshold. The Run#Finished method can be used to determine which runs
+//have arrived at their terminal station, and can therefore be removed from state.
+func (a *RepositoryAgent) GetLatestEstimates(station martaapi.Station) (res []LastestEstimate, err error) {
+	rows, err := a.DB.Query(`
+WITH station_estimates AS (
+  SELECT runs.run_group_identifier,
+    estimates.estimated_arrival_time,
+    estimates.estimate_moment,
```

Comment:
We're only fetching the estimated arrival time itself, the moment when the estimate was made (this is the MARTA "EventTime" field) and the "run group identifier" which we can parse to get the line, direction, and train id.